public/json/option-tab_to_command-tab.json

```
@@ -0,0 +1,33 @@
+{
+  "title": "Bind Option+Tab to Command+Tab ('cycle through applications')",
+  "rules": [
+    {
+      "description": "Bind Option+Tab to Command+Tab",
+      "manipulators": [
+        {
+          "from": {
+            "key_code": "tab",
+            "modifiers": {
+              "mandatory": [
+                "option"
+              ],
+              "optional": [
+                "left_shift"
+              ]
+            }
+          },
+          "to": [
+            {
+              "key_code": "tab",
+              "modifiers": [
+                "left_command"
+              ],
+              "lazy": true
```

```suggestion
              ]
```

AFAIK, `to.lazy` only works if `from.keycode` is a modifier key.

From https://karabiner-elements.pqrs.org/docs/json/complex-modifications-manipulator-definition/to/lazy/:
> If `to.lazy` is `true` and `to.key_code` is modifier flag such as `"key_code": "left_shift"`, the `to.key_code` acts as lazy modifier.

---

Thanks for pointing this out. I didn't fully understand the function of `lazy`. I have removed it now.